src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -314,6 +315,11 @@ public void setMaxRecordsInRam(final int maxRecordsInRam) {
         this.maxRecordsInRam = maxRecordsInRam;
     }
 
+    /** Allows the caller to decide whether or not to write the PG tag to reads when applicable */
```

Comment:
Where would be an appropriate place to document the default value of this property? Maybe here?